internal/operators/ocs/ocs_operator.go

```diff
@@ -103,3 +103,10 @@ func (o *ocsOperator) GetDisksRequirementForMaster(context.Context, *common.Clus
 func (o *ocsOperator) GetDisksRequirementForWorker(context.Context, *common.Cluster) (int64, error) {
 	return 0, nil
 }
+
+// GetProperties provides description of operator properties: none required
+func (o *ocsOperator) GetProperties() models.OperatorProperties {
+	return models.OperatorProperties{
+		OperatorName: o.GetType(),
+	}
```

Comment:
This part should be wrapped by the manager's function `GetOperatorProperties`, and only the array of actual properties be returned by each operator module.